vpr/src/route/rr_node_impl.h

```diff
@@ -123,9 +123,9 @@ inline short t_rr_node::yhigh() const {
     return storage_->node_yhigh(id_);
 }
 
-inline short t_rr_node::capacity() const {
+/*inline short t_rr_node::capacity() const { // no longer used
```

Comment:
Same to my previous comment, we can just remove the line, as a proof that this API is no longer needed

Reply:
Makes sense to remove them to me.

Reply:
They have been removed.